purchase_comment_template/views/purchase_order_view.xml

```diff
@@ -0,0 +1,47 @@
+<odoo>
+
+    <record model="ir.ui.view" id="purchase_order_form_add_comment">
```

Comment:
This needs to be formatted with 4 spaced tabs, there are some formatting issues on the xml files.